SpellCheck/src/main/java/AIR/SpellCheck/Engines/HunspellEngine.java

```diff
@@ -41,23 +42,21 @@ public boolean CheckWord(final String word) {
     final String languageCode = getLanguage();
     boolean isCorrect = true;
     try {
-      if (spellEngineFactory.getDictionary(languageCode).misspelled(word)) {
-        isCorrect = false;
-      }
+      isCorrect = spellEngineFactory.getDictionary(languageCode).isCorrect(word);
```

Comment:
Small API change in the underlying library here.  Rather than a negative ".misspelled" method, there is a positive "isCorrect" method.